### DISCUSSION TITLE
Can't deploy edge functions

### DISCUSSION THREAD

**Discussion by nameisxi** [supabase/supabase#12716] [Questions] [answered]
Hi! I've just started using Supabase's edge functions and was able to deploy them successfully 5 times, until I started to get the following error:
```bash
% supabase functions deploy my-edge-function
Version 1.30.3 is already installed
Bundling my-edge-function
Error: Error bundling function: exit status 1
file:///src/index.ts
file:///_shared/cors.ts
file:///_shared/supabaseClient.ts
error: Uncaught (in promise) Error: Relative import path "events" not prefixed with / or ./ or ../ and not in import map from "https://esm.sh/v108/@types/node@16.18.12/events.d.ts"
      const ret = new Error(getStringFromWasm0(arg0, arg1));
                  ^
    at __wbg_new_8d2af00bc1e329ee (https://deno.land/x/eszip@v0.30.0/eszip_wasm.generated.js:312:19)
    at <anonymous> (https://deno.land/x/eszip@v0.30.0/eszip_wasm_bg.wasm:1:79439)
    at <anonymous> (https://deno.land/x/eszip@v0.30.0/eszip_wasm_bg.wasm:1:1388039)
    at <anonymous> (https://deno.land/x/eszip@v0.30.0/eszip_wasm_bg.wasm:1:1862894)
    at __wbg_adapter_18 (https://deno.land/x/eszip@v0.30.0/eszip_wasm.generated.js:146:6)
    at real (https://deno.land/x/eszip@v0.30.0/eszip_wasm.generated.js:130:14)

Try rerunning the command with --debug to troubleshoot the error.
```

The edge function runs fine locally.

I've tried using an import map (and adding the `https://esm.sh/v108/@types/node@16.18.12/events.d.ts` - whatever it even is - to it) and starting over the project by cloning it into a different directory, but nothing seems to work. Any help would be much appreciated!

**Comment by nameisxi** [comment, marked as answer]
A solution from [this issue](https://github.com/supabase/supabase/issues/10543) worked for me. The package that I was important wasn't Stripe, but by adding `?target=deno&no-check` at the end of the esm.sh link, everything started working.

**Reply by drewbietron** [reply]
This error just started popping up for me as of 3/1/23 for the same Edge Function code that previously deployed without any errors.  This solution worked for me as well ^
